docs/data/osdf/install-cache-container.md

```diff
@@ -115,7 +115,7 @@ This table provides a reminder of the parameters that will be used in the exampl
 |-----------------------|----------------------------------|-----------------------------------------------------------------------------------------|
 | `CACHE_HOST_PORT`     | `8443`                           | The host port used for data transfer via HTTP(S)                                        | 
 | `WEB_INTERFACE_PORT`  | `8444`                           | The host port used for service management via the web interface                         | 
-| `ISSUER_JWK`          | `/etc/osdf-cache/issuer.jwk`     | The private key generated by `pelican generate keygen`                                  | 
+| `PRIVATE_KEY`         | `/etc/osdf-cache/private-key.pem`| The private key generated by `pelican key generate`                                     | 
```

Comment:
Third column has `pelican key generate` but it should be `pelican key create`

Reply:
Fixed